src/App.js

```diff
@@ -1,13 +1,14 @@
 import { Routes, Route, Navigate } from 'react-router-dom';
 import { toMovieDetails, toMoviesList, toPeopleList, toPersonDetails } from "./routes";
 import MoviesList from "./features/moviesList";
+import PeopleList from "./features/peopleList";
 
 function App() {
   return (
     <>
       <Routes>    
           <Route path={toMoviesList()} element={<MoviesList />} />
-          <Route path={toPeopleList()} element="{<PeopleList />}" />
```

Comment:
I think in the future not to add unnecessary extra characters such as quotation marks (even if the component is not yet visible).

Reply:
The quotes were added because at the time of writing this code snippet, the component didn't exist.

If the component is not in the code, the program throws an error and the page will not load in the browser. 

I think this is the simplest way to write this snippet so that another developer knows which component should go there